airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorkerFactory.java

```diff
@@ -92,6 +94,10 @@ public static DefaultReplicationWorker create(final String jobId,
         getStreamFactory(destProtocolVersion, DefaultAirbyteDestination.CONTAINER_LOG_MDC_BUILDER, serdeProvider, migratorFactory),
         new VersionedAirbyteMessageBufferedWriterFactory(serdeProvider, migratorFactory, destProtocolVersion));
 
+    final var srcHeartbeatMonitor = new HeartbeatMonitor(HeartbeatMonitor.DEFAULT_HEARTBEAT_FRESHNESS_THRESHOLD);
```

Comment:
In the future, if we have some sources that we know need a longer heartbeat, we could add a field the actor definition with an override for what the threshold should be.